data/human/intro missions.txt

```diff
@@ -508,3 +508,95 @@ conversation "end of intro missions"
+			choice
+				`	It would be nice to get more information.`
+					goto mining
+				`	I'm busy now, but maybe later.`
+					defer
+				`	I'm not interested.`
+					decline
+	
+			label "mining"
+			`	The warehouse is surprisingly well organized for a place that deals with raw ores. The receptionist tells you that the owner is due back shortly and points you to some picnic tables outside. You have a seat and watch the trucks loading and carting off various minerals. Soon enough a battered Aerie so covered in scorch marks and hull repairs that you can barely make out it's registration markings settles in next to you. When the hatch opens, a woman with close cut hair and a worn but crisp navy flight suit walks up to you. "I'm Captain Zinaida she says without preamble. You were wanting to asking about mining?" You nod your head and she says, "Well, go ahead."`
+			label choices
+			choice
+				`	"Why would I mine rather than run a trading fleet?"`
+					goto whymine
+				`	"What equipment do I need to mine?"`
+					goto equipment
+				`	"Where are safe places to mine?"`
+					goto safety
+				`	"How do you deal with pirates?"`
+					goto pirates
+				`	"Can I mine without and asteroid scanner?"`
+					goto noscanner
+				`	"How can I check the value of minerals?"`
+					goto value
+				`	"Were you in the navy?"`
+					goto navy
+				`	"I can't think of any more questions."`
+					goto end
+			
+			label whymine
+			`	"When you're starting off, mining can be a fast way to build cash for your next ship. It's also a great way to explore the galaxy. Jobs always follow the trade routes and settled areas. Mining makes it profitable to explore the nooks and crannies of the hyperlanes. If the place you end up landing has an economy, they'll buy the minerals in your hold. Mainly though, it's about how you like to spend your time. Sure, you can get rich running a trading fleet and managing other captains, but miners get to fly their ship directly." The look she gives you is both assessing and challenging.`
+				goto more
+			
+			label equipment
+			`	Captain Zinaida looks over the <ship> critically. "Well, you need a weapon that does hull damage. A beam laser will get the job done but it will take awhile. The most valuable tool is an asteroid scanner. They're expensive but they connect into your navigation and targeting systems to point you towards minable asteroids. You can also set them to find the most valuable asteroids which increases your profitability per run. After that, a tactical scanner will give you range to target and show how much damage you're doing. An afterburner is good for chasing down fast moving asteroids or getting away from pirates without leaving the system. If you have a fleet, you can target an asteroid and they will all fire on it and then pick up the minerals. You can work much more quickly and harvest more in a run. If you have a carrier, like Bertha there, you can have fighters mine and return the minerals to your hold.`
```

Comment:
```suggestion
			`	Captain Zinaida looks over the <ship> critically. "Well, you need a weapon that does hull damage. A beam laser will get the job done but it will take awhile. The most valuable tool is an asteroid scanner. They're expensive but they connect into your targeting systems to show you where mineable asteroids are. You can also set them to find the most valuable asteroids which increases your profitability per run. After that, a tactical scanner will give you range to target and show how much damage you're doing. An afterburner is good for chasing down fast moving asteroids or getting away from pirates without leaving the system. If you have a fleet, you can target an asteroid and they will all fire on it and then pick up the minerals. You can work much more quickly and harvest more in a run. If you have a carrier, like Bertha there, you can have fighters mine and return the minerals to your hold.`
```
This made it sound like you always had the pointing towards target setting on.